src/components/pages/AddStaff/AddStaff.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useState } from 'react';
+import {
+  registrationHandler,
+  getToken
+} from '../../../utils/handlers/authenticationHandlers';
+import AddStaffForm from './AddStaffForm';
+import validateAddStaffForm from './AddStaffValidation';
+
+function AddStaff() {
+  const [state, setState] = useState({
```

Comment:
❓ Can you rename the `setState` function? It is confusing because `setState` is a function within React. 
Maybe call it `updateState` ?